reputation/reputation_contract.go

```diff
@@ -94,12 +94,7 @@ func Put(epoch int, peerID []byte, value []byte) {
 	ctx := storage.GetContext()
 
 	multiaddr := common.AlphabetAddress()
-	alphabetCall := runtime.CheckWitness(multiaddr)
-
-	if !alphabetCall {
-		runtime.Notify("reputationPut", epoch, peerID, value)
-		return
-	}
+	common.CheckAlphabetWitness(multiaddr)
```

Comment:
BTW, the behaviour has been changed, is it OK? The previous behaviour was to return from the function if not witnessed by alphabet, and the current one is to panic.

Reply:
Yes, this is explained in the commit message.

Reply:
Oh, sorry, I've checked the "Files changed" section only.